src/transformers/pipelines/__init__.py

```diff
@@ -528,7 +528,7 @@ def pipeline(
     load_tokenizer = type(model_config) in TOKENIZER_MAPPING or model_config.tokenizer_class is not None
     load_feature_extractor = type(model_config) in FEATURE_EXTRACTOR_MAPPING or feature_extractor is not None
 
-    if task in {"audio-classification"}:
+    if task in {"audio-classification", "image-classification"}:
```

Comment:
I would update the comment below in case "image-classification" also doesn't require a tokenizer.

Reply:
Thanks !